clang/test/Driver/sycl-offload-intelfpga.cpp

```diff
@@ -320,9 +320,9 @@
 /// -fintelfpga output dep file using -Fo<dir>
```

Comment:
This test is actually not valid anymore.  It was to catch problems of the dep name when using /Fo.  Since it is all temporary now I can clean this up.